quadrat/sass/templates/_header.scss

```diff
@@ -31,6 +31,15 @@
 		padding-right: 0;
 	}
 
+	.wp-block-site-tagline {
+		margin: 0;
+		@include break-mobile-only() {
```

Comment:
Could we do this with intrinsic CSS, so that it wraps when it needs to rather than at a specific point?

Reply:
I tinkered with that, and agree that would be desirable, but I wasn't able to figure out how.  It's not just a matter of wrapping, but the design is a bit different in the two scenarios.  The Navigation block is already switching to mobile design (using the same Gutenberg breakpoint) so it doesn't feel wrong.

Reply:
Actually... upon further evaluation I see that the navigation block changes styles at SMALL rather than MOBILE breakpoint... Refactoring slightly to break at the same point for the rest of the quadrat header.